cmd/dex/serve.go

```diff
@@ -145,6 +149,8 @@ func serve(cmd *cobra.Command, args []string) error {
 		s = storage.WithStaticPasswords(s, passwords)
 	}
 
+	serverLogger := logger.WithField("unit", "server")
```

Comment:
don't know if unit is good since systemd already has a "unit". Maybe we just drop this for now?